client/client_test.go

```diff
@@ -155,3 +155,19 @@ func (cs *clientSuite) TestClientCapabilities(c *check.C) {
 		},
 	})
 }
+
+func (cs *clientSuite) TestClientAddCapability(c *check.C) {
+	cs.rsp = `{
+		"type": "sync",
+		"result": {
+		}
+	}`
+	cap := &client.Capability{
+		Name:  "n",
+		Label: "l",
+		Type:  "t",
+		Attrs: map[string]string{"k": "v"},
+	}
+	err := cs.cli.AddCapability(cap)
```

Comment:
I see your point. Since the method returns nil on success I find it hard to write a compelling test. I'm open to suggestions.